platform-metrics/build.gradle.kts

```diff
@@ -27,6 +27,8 @@ dependencies {
   implementation("org.eclipse.jetty:jetty-servlet:9.4.53.v20231009")
   implementation("com.google.guava:guava:32.0.1-jre")
 
+  compileOnly("com.github.ben-manes.caffeine:caffeine:3.1.8")
```

Comment:
hmm - test is failing I suspect because this is an overload with the same name as the guava cache, so needed at runtime to find the correct signature. Maybe change the method name to `registerCaffeineCache*`?

Reply:
yeah - am renaming this and writing separate test for this.. will push soon..